lib/vagrant-libvirt/action/destroy_domain.rb

```diff
@@ -75,6 +75,14 @@ def call(env)
             root_disk.destroy if root_disk
           end
 
+          # Remove NFS exports that are no longer valid following destroy
+          uuid = env[:machine].id
+          uuids = env[:machine].provider.driver.connection.servers.all.map(&:id)
+          uuids.delete(uuid)
+          env[:host].capability(
+            :nfs_prune, env[:machine].ui, uuids
+          )
+
```

Comment:
This seems like we should simply be adding back in the action PruneNFSExports to the self.action_destroy method in lib/vagrant-libvirt/action.rb.